### PR TITLE
Include learning transport in SCMU

### PR DESCRIPTION
**Pull request by jpalac**
Include the LearningTransport in SCMU if Environment variable ServiceControl_IncludeLearningTransport is true

### CODE REVIEW

File: src/ServiceControlInstaller.Engine/Instances/TransportNames.cs [hunk 1]
```diff
@@ -37,6 +37,7 @@ public static class TransportNames
         public const string RabbitMQClassicDirectRoutingTopology = "RabbitMQ - Direct routing topology (classic queues)";
 
         public const string RabbitMQQuorumDirectRoutingTopology = "RabbitMQ - Direct routing topology (quorum queues)";
+        public const string LearningTransport = "Learning Transport";
```
Comment (mikeminutillo) [inline src/ServiceControlInstaller.Engine/Instances/TransportNames.cs:40]: None of the others include transport in the name

```suggestion
        public const string LearningTransport = "Learning";
```

**Review by mikeminutillo — APPROVED** [review]

**Review by mauroservienti — APPROVED** [review]

**Comment by mauroservienti** [comment]
Do we want to document this in the README?

**Comment by jpalac** [comment]
> Do we want to document this in the README?

Yes but I'd like to test this first - would actually like to see this in the installer. I have no way of creating the installer locally, so the only way might be to push an alpha on this branch?...

**Comment by mauroservienti** [comment]
I think you can also just F5 the SCMU project, it should work. Not sure it can install instances, thought.

**Comment by jpalac** [comment]
Ok I've verified that it loads the option into the dropdown if the env variable is present. It also created the instance with the "queues" being created in the file system - guess it's ready for smoke testing

**Pull request merged**